python/lsst/ts/observatory/control/mock/mtcs_async_mock.py

```diff
@@ -461,6 +470,33 @@ async def mtdome_tel_light_wind_screen_next(
     ) -> types.SimpleNamespace:
         return self._mtdome_tel_light_wind_screen
 
+    async def mtdome_cmd_park(self, *args: typing.Any, **kwargs: typing.Any) -> None:
+        asyncio.create_task(self._mtdome_park())
+
+    async def _mtdome_park(self, *args: typing.Any, **kwargs: typing.Any) -> None:
```

Comment:
you are not passing anything to this method so you can probably remove the arguments.

Reply:
592b7c8c01531a548501928ab2d23e4932d44244